cargo-dist/src/backend/ci/github.rs

```diff
@@ -217,10 +243,68 @@ impl GithubCiInfo {
         Ok(rendered)
     }
 
+    /// Generate and write the vendored actions to disk
+    pub fn write_vendored_actions(&self, dist: &DistGraph) -> DistResult<()> {
+        if dist.tools.git.is_none() {
+            warn!("Unable to vendor external actions; git is missing");
+            return Ok(());
+        }
+
+        let path = dist.workspace_dir.join(GITHUB_ACTIONS_DIR);
+
+        vendor_repo(
```

Comment:
Yeah, I was wondering what the nicest way to structure this might be. Limited somewhat by the fact that some of the structures I'd reach for to keep this organize can't be static/const at the top of the file like those constant strings are.

Reply:
Updated to have a method that formats these into a vec of organized tuples.